Changes

```diff
@@ -285,6 +285,9 @@ _______________
   (Jan Midtgaard, review by Antonin Décimo, Sébastien Hinderer, and
    David Allsopp)
 
+- #13185: Type-level module alias for functor parameter accepted
```

Comment:
The changes entry might mislead readers that missed the fact that we are describing the bug and not the fix.
Maybe  `reject type-level module aliases on functor parameters`?